server.js

```diff
@@ -9,11 +9,25 @@
 
 require('babel-core/register');
 
+var optional = require("optional");
 var bl = require('bl');
 var express = require('express');
 var mentionBot = require('./mention-bot.js');
 var GitHubApi = require('github');
 var config = require('./package.json').config;
+var util = require('util');
+var fs = require('fs');
+var configuredMessageGenerator = optional("./message.js");
+var defaultMessageGenerator = function(reviewers) {
+  return util.format("By analyzing the blame information on this pull request" + 
```

Comment:
nit: please only indent by two spaces, use single quote for strings and no need to put parenthesis around the conditional anymore:

```javascript
return util.format(
  'By analyzing the blame information on this pull request' + 
  ', we identified %s to be%s potential reviewer%s',
  buildMentionSentence(reviewers),
  reviewers.length > 1 ? '' : ' a', 
  reviewers.length > 1 ? 's' : ''
);
```